index.css

```diff
@@ -1 +1,1091 @@
+  left: 2%;
+  margin-top: 20px;
+  text-align: center;
+}
+
+.hed-2 > h2 {
+  width: 95%;
+  font-family: 'Lato', sans-serif;
+  font-weight: 300;
+  font-size: 28px;
+  color: rgb(239, 81, 60);
+}
+
+.hed-2 > h1 {
+  width: 100%;
+  font-family: 'Lato', sans-serif;
+  font-weight: 1000;
+  font-size: 40px;
+  color: var(--red);
+}
+
+.conf2 {
+  width: 95%;
+  position: relative;
+  border: 4px solid rgb(247, 247, 248);
+  background-color: rgb(255, 255, 255);
+  box-sizing: border-box;
+  text-align: center;
+  left: 5%;
+}
+
+.conf2 > p {
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  color: var(--black);
+  word-break: keep-all;
+  position: relative;
+  margin: 15px;
+  padding-top: 15px;
+  padding-bottom: 15px;
+}
+
+.header-pag > p,
+h4 {
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  padding-top: 20px;
+}
+
+.header-pag > h4 {
+  padding-top: 0;
+  padding-bottom: 35px;
+}
+
+.logo {
+  padding-top: 30px;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  width: 80%;
+  position: relative;
+  left: 10%;
+}
+
+.logo > h2 {
+  width: 80%;
+  position: relative;
+  left: 10%;
+}
+
+.logo > img {
+  width: 7%;
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+.logo > p {
+  width: 90%;
+  position: relative;
+  left: 5%;
+  padding-bottom: 30px;
+}
+
+.logo2 {
+  width: 95%;
+  position: relative;
+  border: 4px solid rgb(247, 247, 248);
+  background-color: rgb(255, 255, 255);
+  box-sizing: border-box;
+  text-align: center;
+  left: 2%;
+  padding: 5px;
+}
+
+#logo {
+  margin: 10px;
+  width: 30%;
+}
+
+.past {
+  margin-top: 40px;
+  padding-top: 30px;
+  text-align: center;
+  border: 4px solid rgb(247, 247, 248);
+  background-color: rgb(255, 255, 255);
+}
+
+.past > img {
+  width: 7%;
+}
+
+.past > h2 {
+  font-size: 28px;
+  font-family: 'Lato', sans-serif;
+  font-weight: 600;
+}
+
+.past > p {
+  font-family: 'Lato', sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  width: 90%;
+  padding-top: 10px;
+  position: relative;
+  left: 5%;
+}
+
+.pss1 {
+  background-image: url("./images/about_201.png");
+  background-repeat: no-repeat;
+  background-color: var(--black);
+  height: 250px;
+  width: auto;
+  max-width: 480px;
+  box-shadow: 0 2px 8px 0 rgb(0 0 0 / 30%);
+}
+
+.pss2 {
+  background-image: url("./images/about_203.png");
+  background-repeat: no-repeat;
+  height: 250px;
+  width: auto;
+  max-width: 480px;
+  box-shadow: 0 2px 8px 0 rgb(0 0 0 / 30%);
+}
+
+.psss {
+  display: grid;
+  grid-template-columns: 80%;
+  gap: 5%;
+  padding: 5px;
+  position: relative;
+  left: 8%;
+}
+
+.pss1 > h3,
+.pss2 > h3 {
+  color: rgb(238, 237, 237);
+  font-family: 'Lato', sans-serif;
+  font-size: 33px;
+  font-weight: 300;
+  position: relative;
+  top: 45px;
+}
+
+.pss1 > p,
+.pss2 > p {
+  color: rgb(207, 173, 173);
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  position: relative;
+  top: 15px;
+}
+
+@media (min-width: 768px) {
```

Comment:
Great job on using `media-queries` 😄 , it shows how much you understand design and cascading stylesheets (CSS) 💧  but maybe you need to use more of them or adjust 📏  what you currently have because the design is breaking 🤕 when I try to resize. See below 👇🏽 
- Here you can see the `PSS-Campaign` button getting pushed out of the `navbar` as well as the `text` I asked to remove earlier, falling below the logo.
![Screenshot 2023-02-09 at 07 42 20](https://user-images.githubusercontent.com/45739963/217816009-ef2f247a-dc02-4616-a3ff-dd83388f0b9f.png)

- Here you can see the extra horizontal space existing in the web app. See below 👇🏽 
![Screenshot 2023-02-09 at 07 45 10](https://user-images.githubusercontent.com/45739963/217816378-44443899-f958-4520-b0b1-eeda53c9066f.png)

In order to fix that you need to add this line to your `body` in CSS:
```
overflow-x: hidden;
```
Check this [link](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow) for more information on `overflow` 🔗